From 0e8e89b19a4e47802ed6ea64a35da77665cc3d3f Mon Sep 17 00:00:00 2001
From: Paul Kocialkowski <contact@paulk.fr>
Date: Mon, 10 Aug 2015 20:33:23 +0200
Subject: [PATCH 1/7] firmware: Developer mode timeout delay shortening (down
 to 3 seconds)

A timeout delay of 3 seconds, with no bip, is much more appreciable for users.

Change-Id: If4bc879f350a4a0297395077e4f55e89f4fa764d
Signed-off-by: Paul Kocialkowski <contact@paulk.fr>
---
 firmware/lib/vboot_audio.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/firmware/lib/vboot_audio.c b/firmware/lib/vboot_audio.c
index 8f31171..7e92dc7 100644
--- a/firmware/lib/vboot_audio.c
+++ b/firmware/lib/vboot_audio.c
@@ -30,11 +30,7 @@
 #define MAX_CUSTOM_DELAY      300000
 
 /* These are visible externally only to make testing easier */
-VbDevMusicNote default_notes_[] = { {20000, 0}, /* 20 seconds */
-                                    {250, 400}, /* two beeps */
-                                    {250, 0},
-                                    {250, 400},
-                                    {9250, 0} }; /* total 30 seconds */
+VbDevMusicNote default_notes_[] = { {3000, 0} }; /* three seconds */
 uint32_t default_count_ = sizeof(default_notes_) / sizeof(VbDevMusicNote);
 
 VbDevMusicNote short_notes_[] = { {2000, 0} }; /* two seconds */
-- 
2.10.2

